utils/fo-installdeps

```diff
@@ -85,6 +107,31 @@ CODENAME=$(lsb_release --codename --short)
 
 echo "install core dependencies"
 
+# Darwin
+if [[ $BUILDTIME && "$(uname)" == "Darwin" ]]; then
```

Comment:
Would recommend creating a new variable like `CODENAME` and `DISTRO` for `uname`. Something like `ARCH` sounds good?

Reply:
In mac, is there any equivalent for `lsb_release`? If not, the script will break at line 103 and 104.

Reply:
No, Lsb_release is not available in macOs. instead I'm trying to opt for `sw_vers -productVersion`, while creating a new varible like `codename`, `Distro` & `Arch`